_data/downloads.yml

```diff
@@ -1,29 +1,24 @@
 stepOne:
   - os: linux
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <div class="highlight">
+        <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup </code> <br>
+      </div>
   - os: unix
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+      <div class="highlight">
+        <p><code class="hljs">$ ./cs setup</code></p>
+      </div>
   - os: macos
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <div class="highlight">
+        <code class="hljs">$ brew install coursier/formulas/coursier && cs setup </code> <br>
```

Comment:
The current download page links to the official sbt page to setup sbt. I am concerned about `cs setup` because it would install a fake application called `sbt` that is actually not the Bash script maintained by sbt team. This end up creating needless confusion because inevitably some behavior will be different and the user would complain to the sbt project, not Couriser. See for example https://github.com/sbt/sbt/issues/6299.

Reply:
Would it be possible to unify them? @alexarchambault

Reply:
seconded that the coursier-based sbt launcher is not yet ready for this level of exposure